test.py

```diff
@@ -14,11 +14,11 @@ def test_mother(self):
         user = '__'
         tweets = list(get_tweets(query=user, pages=1))
 
-        self.assertTrue(tweets[0]['text'].__contains__('It is a gift to be alive in the time of Beyoncé'))
+        self.assertTrue(tweets[3]['text'].__contains__('It is a gift to be alive in the time of Beyoncé'))
 
     def test_child(self):
         user = '___'
-        tweets = list(get_tweets(query=user, pages=1))
+        tweets = list        (get_tweets(query=user, pages=1))
```

Comment:
Why you leave a tab between **list** and **(** ?

Reply:
Beginner's error. This was my first pull request and I made a goof while editing.

Reply:
It's okay, not need new one.